app/controllers/topological_inventory/ingress_api/v0/inventory_controller.rb

```diff
@@ -0,0 +1,33 @@
+module TopologicalInventory
+  module IngressApi
+    module V0
+      class InventoryController < ApplicationController
+        skip_before_action :verify_authenticity_token
```

Comment:
Not sure why this is needed since [`ApplicationController < ActionController::Api`](https://github.com/ManageIQ/topological_inventory-ingress_api/pull/5/files#diff-55c5b7aecfb519d0e4880eaf2788eb6eR1) but without I'm getting:

```
Can't verify CSRF token authenticity.
Completed 422 Unprocessable Entity in 0ms (ActiveRecord: 0.0ms)
```

on POST